src/intent.html

```diff
@@ -295,16 +295,17 @@ <h3 id="mixing_intent_properties">Intent Properties</h3>
    <h3 id="intent_using">Using Intent Concepts and Properties</h3>
    <p>When the <code class="attribute">intent</code> attribute corresponding to a specific node
      contains a concept component, possibly with properties and arguments,
-     the AT's [=Intent Concept Dictionary=] should be consulted.
+     the supported [=Intent Concept Dictionary=] entries should be consulted.
      The concept name should be normalized
      (<q>`_`</q> (U+00F5) and <q>`.`</q>  (U+002E) to <q>`-`</q> (U+002D)),
      and compared using <a data-cite="INFRA#ascii-case-insensitive">ASCII case-insensitive</a>
      match.</p>
-   <p>An concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
-     when the normalized name,
-     the fixity property (which may be defaulted in the concept dictionary),
+   <p>A concept is considered a <dfn id="intent_known_concept">known concept</dfn> (to the AT)
+     when the AT has any internal behavior recognizing that concept.
+     It is also considered a <dfn id="intent_matched_concept"></dfn>matched concept</dfn> when
+     its normalized name, the fixity property (which may be defaulted in the concept dictionary),
      and the arity (number of arguments, if any)
-     all match an entry in the AT's concept dictionary.
+     all match an entry in the Core or Open concept dictionaries.
```

Comment:
No sorry I think this a complete change in the meaning of the core and open lists. There has never been any expectation that a system reads the open list, it is just guidance for implementors on names to use. Thus the classification of a concept as known/unknown (or matched/unmatched) which affects its fallback rendering as a literal, can not depend on the open list, it has to, as in the current draft, and Bruce's PR, depend on the system-specific names that are matched by the AT in use.